packages/webpack-utilities/package.json

```diff
@@ -16,11 +16,10 @@
   },
   "devDependencies": {
     "@fluentui/eslint-plugin": "^1.3.3",
-    "@fluentui/scripts": "^1.0.0",
-    "@types/loader-utils": "1.1.3"
+    "@fluentui/scripts": "^1.0.0"
   },
   "dependencies": {
-    "loader-utils": "^1.1.0",
+    "loader-utils": "^2.0.0",
```

Comment:
According to the [`CHANGELOG.md`](https://github.com/webpack/loader-utils/blob/master/CHANGELOG.md) this safe bump, however this package does not have tests at all so I can't be sure.